utility/importer/brave_profile_import_impl.cc

```diff
@@ -23,11 +59,54 @@ void BraveProfileImportImpl::StartImport(
   } else if (base::StartsWith(base::UTF16ToUTF8(source_profile.importer_name),
                               "Chromium", base::CompareCase::SENSITIVE)) {
     command_line->AppendSwitch("import-chromium");
-  } else if (base::StartsWith(base::UTF16ToUTF8(source_profile.importer_name),
-                              "Brave", base::CompareCase::SENSITIVE)) {
-    command_line->AppendSwitch("import-brave");
   }
 
-  ProfileImportImpl::StartImport(
-      source_profile, items, localized_strings, std::move(observer));
+  content::UtilityThread::Get()->EnsureBlinkInitialized();
+  importer_ = CreateImporterByType(source_profile.importer_type);
+  if (!importer_.get()) {
+    mojo::Remote<chrome::mojom::ProfileImportObserver>(std::move(observer))
```

Comment:
actually we should also override `OnImportFinished` then we don't even have to bind `chrome::mojom::ProfileImportObserver` and then https://github.com/brave/brave-core/pull/5582/commits/0494a09ac517dbbea74a7fda9170916ac1985599#r441900049 wouldn't be necessary

Reply:
resolved by https://github.com/brave/brave-core/pull/5582#discussion_r441908478